File: DrivingNote/ClientApp/src/Models/UserInfo.ts
import { Cell } from "./Cell";

export default class UserInfo {
  Name: string;
  LastName: string;
  Email: string;
  Sport: string;
  AccountNumber: string;
  HTMLTable: Cell[];

  constructor(name: string, lastname: string, email: string, sport: string, accountNumber: string, htmlTable: Cell[]) {
    this.Name = name;
    this.LastName = lastname;
    this.Email = email;
    this.Sport = sport;
    this.AccountNumber = accountNumber;
    this.HTMLTable = htmlTable;
  }
}

File: DrivingNote/Models/UserInfo.cs
using System.Collections.Generic;

namespace DrivingNote.Models
{
    public class UserInfo
    {
        public string Name { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string Sport { get; set; }
        public string AccountNumber { get; set; }
        public List<HTMLTable> HTMLTable { get; set; }
    }
}
